torchmultimodal/utils/common.py

```diff
@@ -24,16 +24,21 @@ def get_current_device() -> Union[str, torch.device]:
 def get_extended_attention_mask(attention_mask: Tensor) -> Tensor:
     """
     Makes broadcastable attention and causal masks so that future and masked tokens are ignored.
+    Arguments:
+        attention_mask (`torch.Tensor`):
+            Mask with ones indicating tokens to attend to, zeros for tokens to ignore.
 
-    Args:
-        attention_mask (Tensor): Mask with ones indicating tokens to attend to, zeros for tokens to ignore.
     Returns:
-        extended_attention_mask (Tensor): extended attention mask with the same dtype as attention_mask.dtype.
+        `torch.Tensor` The extended attention mask, with the same dtype as `attention_mask.dtype`.
```

Comment:
Double back tick for code sample inline expansion. Single back tick has reserved meaning in reStructuredText.
"Tensor" instead of "torch.Tensor"?